pom.xml

```diff
@@ -68,8 +69,10 @@
         <maven-deploy-plugin.version>3.1.1</maven-deploy-plugin.version>
         <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
         <parquet.version>1.12.2</parquet.version>
-        <scala.version>2.12.15</scala.version>
-        <scala.version.prefix>2.12</scala.version.prefix>
+        <scala12.version>2.12.15</scala12.version>
+        <scala13.version>2.13.8</scala13.version>
```

Comment:
why not use Scala 2.12.20 and 2.13.14 - the latest versions of these patch lines?

Reply:
Based on the Spark version, we need to use a supported Scala version. Specifically, **Spark 3.4** version supports **Scala 2.12.17** for **Scala 2.12** and **Scala 2.13.8** for **Scala 2.13**.

Whenever I address all review comments, I will update the Scala 2.12 version to 2.12.17.

**Reference:**

1. https://github.com/apache/spark/blob/branch-3.4/pom.xml#L3624

Reply:
You should be able to use the latest patch level Scala compiler releases. This is what the Scala team recommend.